internal/middleware/cors.go

```diff
@@ -7,8 +7,7 @@ import (
 
 // Updated enableCors function to return a boolean indicating if OPTIONS was handled
 func handleCors(w http.ResponseWriter, r *http.Request) bool {
-	configValue := config.LoadConfig()
-	allAllowedOrigins := configValue.Server.AllowedOrigins
+	allAllowedOrigins := config.AppConfig.Server.AllowedOrigins
```

Comment:
Not sure why linter is failing here even though the config.LoadConfig is changed here..getting linter error where this function is called in internal/middleware/ratelimiter.go